sympy/functions/elementary/hyperbolic.py

```diff
@@ -1540,6 +1543,9 @@ def _eval_is_zero(self):
         if (self.args[0] - 1).is_zero:
             return True
 
+    def _eval_is_real(self):
+        return self.args[0].is_real and (self.args[0] - 1).is_positive
```

Comment:
Potentially checking `is_real` here is redundant because the `is_positive` query would be False if `is_real` is False. I prefer not to create the expression `arg - 1` though so it is useful to have that shortcut.